### PR TITLE
Kraken : Fix Address Label in POI

### PR DESCRIPTION
**Pull request by mbounabi**
https://jira.canaltp.fr/browse/NAVITIAII-2349

### CODE REVIEW

File: source/type/pb_converter.cpp [hunk 1]
```diff
@@ -1165,7 +1165,7 @@ void PbCreator::Filler::fill_pb_object(const ng::POI* poi, pbnavitia::Address* a
         address->set_house_number(poi->address_number);
         label += std::to_string(poi->address_number) + " ";
     }
-    label += get_label(poi);
+    label += poi->address_name;
```
Comment (xlqian) [inline source/type/pb_converter.cpp:1168]: @kinnou02

Comment (kinnou02) [reply source/type/pb_converter.cpp:1168]: https://github.com/mbounabi/navitia/blame/6ddd03d8d37582154021b6b786f1871c339aff17/source/type/pb_converter.cpp#L2278
regression introduced by https://github.com/mbounabi/navitia/commit/3b561bdb0fb3af34c7f5dd7ef90c8a4e91144707

**Comment by mbounabi** [comment]
jenkins, test this please!

**Comment by xlqian** [comment]
Jenkins, test this please!

**Comment by xlqian** [comment]
Jenkins, test this please!

**Comment by xlqian** [comment]
Jenkins, test this please!

**Review by xlqian — APPROVED** [review]

**Review by xlqian — APPROVED** [review]

**Review by benoit-bst — APPROVED** [review]

**Pull request merged**